query/src/main/java/org/apache/accumulo/examples/wikisearch/iterator/DefaultIteratorEnvironment.java

```diff
@@ -18,30 +18,24 @@
 
 import java.io.IOException;
 
-import org.apache.accumulo.core.client.sample.SamplerConfiguration;
+import org.apache.accumulo.core.client.impl.BaseIteratorEnvironment;
```

Comment:
Bad form to depend on an impl class. There aren't any API guarantees on BaseIteratorEnvironment not changing in a way that would break DefaultIteratorEnvironment. Would be better to leave this as it was (implements IteratorEnvironment).